src/command/http-command.ts

```diff
@@ -142,6 +168,21 @@ export class HttpCommand implements CommandInterface<HttpOptions> {
 			resolve();
 		};
 
+		const captureCert = (socket: TLSSocket): Cert | undefined => ({
+			...socket.getPeerCertificate(),
+			authorized: socket.authorized,
+			authorizationError: socket.authorizationError,
+		});
+
+		// HTTPS cert is not guaranteed to be available after this point
```

Comment:
So just to verify, we disable the cache, but it still may not be available at the end and we need to move this here?

Reply:
That's the correct way to do it - to capture the cert while socket is still connected. By the time the response comes, the socket should be already disconnected, and hence won't have the TLS record.

```
Returns an object representing the peer's certificate. If the peer does not provide a certificate, an empty object will be returned. If the socket has been destroyed, null will be returned.
```
https://nodejs.org/api/tls.html#tlssocketgetpeercertificatedetailed